css/donate.css

```diff
@@ -49,31 +56,42 @@ h2 {
 }
 
 .story-image {
-  padding-right: 50px;
   padding-bottom: 50px;
 }
 
 .donate-promise {
   background-color: #E4D7ED;
 }
 
+.our-donor {
+  padding-top: 50px;
+  text-align: center;
+}
+
 .promise-header {
   padding-top: 50px;
-  position: relative;
+  text-align: center;
 }
 
 .promise-image {
-  padding-bottom: 50px;
+  padding-top: 50px;
```

Comment:
this looks fine until the columns collapse vertically, at which there is some awkward empty space above the image. can you add something in the media query to change the padding-top for this to 0?